test/example/c_api/CMakeLists.txt

```diff
@@ -1,14 +1,16 @@
-cmake_minimum_required(VERSION 3.20...3.29)
+cmake_minimum_required(VERSION 3.25...3.29)
 project(example_spglib LANGUAGES C)
 
-set(CMAKE_C_STANDARD 11)
+set(CMAKE_C_STANDARD 17)
 set(CMAKE_C_STANDARD_REQUIRED ON)
 set(CMAKE_C_EXTENSIONS OFF)
 
 find_package(Spglib REQUIRED)
 
 add_executable(example example.c)
-add_executable(example_full example_full.c)
-
 target_link_libraries(example PRIVATE Spglib::symspg)
-target_link_libraries(example_full PRIVATE Spglib::symspg)
+
+if (UNIX)
+	add_executable(example_full example_full.c)
+	target_link_libraries(example_full PRIVATE Spglib::symspg)
+endif ()
```

Comment:
Why does this not compile under windows?

Reply:
That issue I haven't managed to debug. This is avoidable by making it compile-time, (for some reason windows does not) or using `malloc`. One thing I haven't tested is if it's a C standard thing or if it's a flag/optimization thing